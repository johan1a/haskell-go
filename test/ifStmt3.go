func main() {
if 2 == 2 {
    print( 1 )
} 
}

