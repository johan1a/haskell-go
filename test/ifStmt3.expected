[TopLevelDecl2 (FunctionDecl2 "main" Signature1 (Block [IfStmt (Ifstmt1 (BinExpr (CondExpr (Eq_ (Num 2) (Num 2)))) (Block [SimpleStmt (ExpressionStmt (PrintCall [Num 1]))]))]))]
